pwnlib/timeout.py

```diff
@@ -96,15 +96,15 @@ class Timeout(object):
         0.1
         >>> print(t.timeout)
         5.0
-        >>> with t.local(0.5):
+        >>> with t.local(0.5):# doctest: +ELLIPSIS
         ...     for i in range(5):
         ...         print(round(t.timeout,1))
         ...         time.sleep(0.1)
         0.5
         0.5
         0.5
         0.5
-        0.5
+        ...
```

Comment:
I like it, not depending on timings always makes me feel warm

Reply:
I think this actually caused failures for some reason and was reverted